### ISSUE TITLE
Include does not work when including same file twice in different files.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [unlockha/beanio#84]
```
I have a main XML configuration file streams.xml. I have a common template file 
template.xml. Finally, I have two mapping files a.xml and b.xml describing 
configuration for class A and B.

The layout is the following:
streams.xml includes a.xml and b.xml
a.xml and b.xml include templates.xml

Both the stream of A and B use the same template. The stream factory finds the 
template the first time but not the second.

As a workaround, I copy/pasted the templates in b.xml.
```

Original issue reported on code.google.com by `fbar...@scanntech.com` on 18 Jun 2013 at 10:00

**Comment by GoogleCodeExporter** [comment]
```
If you let me know what version you are using, I can post a snapshot JAR here.

Thanks,
Kevin
```

Original comment by `kevin.s...@gmail.com` on 19 Jun 2013 at 2:32
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Hi, I'm using 2.0.6.

Thanks.
```

Original comment by `fbar...@scanntech.com` on 19 Jun 2013 at 1:45

**Comment by GoogleCodeExporter** [comment]
```
Fixed for 2.0.7.  Snapshot JAR attached.
```

Original comment by `kevin.s...@gmail.com` on 20 Jun 2013 at 2:29
- Added labels: **Milestone-2.0.7**

Attachments:
- [beanio.jar](https://storage.googleapis.com/google-code-attachments/beanio/issue-84/comment-3/beanio.jar)

**Comment by GoogleCodeExporter** [comment]
Original comment by `kevin.s...@gmail.com` on 7 Sep 2013 at 6:19
- Changed state: **Fixed**
